.changeset/fixanalyzer_suppression_comment_fails_with_inner_comments_in_functions.md

```diff
@@ -0,0 +1,17 @@
+---
```

Comment:
You must add the cli package too. Please read the contribution guide regarding the changesets

Reply:
done